src/main/java/calculator/AddCalculator.java

```diff
@@ -0,0 +1,66 @@
+package calculator;
+
+import calculator.exception.NegativeNumException;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class AddCalculator {
+
+    public static int execute(String str) {
+        if (isInValidString(str))
+            return 0;
+
+        String[] arr = splitString(str);
+
+        return getSum(arr);
+    }
+
+    private static String[] splitString(String str) {
+        String[] arr = splitWithCustomDelimiter(str);
+
+        if (arr != null) {
+            return arr;
+        }
+
+        return str.split(":|,");
+    }
+
+    private static String[] splitWithCustomDelimiter(String str) {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(str);
+
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            return m.group(2).split(customDelimiter);
+        }
+
+        return null;
+    }
+
+    private static boolean isInValidString(String str) {
+        if (str == null || str.equals(""))
```

Comment:
```suggestion
       return str == null || str.equals("")
```
으로 좀 더 간단히 표현해볼수도 있을거 같습니다 ㅎㅎ